query.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require 'pg'
+require 'dotenv/load'
+
+CONNECTION = PG.connect(
+  dbname: ENV['DB_NAME'],
+  user: ENV['DB_USER'],
+  password: ENV['DB_PASSWORD']
+)
+
+def select_all
+  CONNECTION.exec(<<~SQL)
+    SELECT *
+    FROM memos
+    ORDER BY id ASC;
```

Comment:
```suggestion
    ORDER BY id ASC
```

psqlなどと違い、クエリは即実行されるので`;`は付けないことが多いですね